navis/morpho/mmetrics.py

```diff
@@ -30,6 +30,12 @@
 # Set up logging
 logger = config.get_logger(__name__)
 
+# Set up numpy number representation, see NEP51
+# Once numpy<=2 is dropped from requirements, the doctest comparissons
+# should become `np.float64(1.074)` instead of `1.074`
+if int(np.version.version.split('.')[0])>=2:
```

Comment:
I made the suggested changes.

In this scenario, the whole code basis gets switched to a different representation of np values. This means, in theory there could a piece of code that behaves differently in the `NAVIS_TEST_ENV` than it does whenever you run the code without that variable set to true. I can't think of a case where this would matter. Since you were worried about "global" changes to the representation / print option, I also made an alternative PR #177, where the change of number representation should only change within the pydoctest.

Either #176 or #177 should be accepted, not both.

Reply:
Thanks! I think I prefer the solution in this PR over #177 (the latter feels more clunky).